test/unit_tests/apps/financial-well-being/fwb-questions.js

```diff
@@ -0,0 +1,169 @@
+                <fieldset class="o-form_fieldset">
+                    <h3 class="o-scale_header">
+                        Select how you completed the questionnaire. This changes the scoring calculation.
+                    </h3>
+                    <ul class="m-list m-list__unstyled content-l">
+                        <li class="content-l_col content-l_col-1-2">
+                            <div class="m-form-field m-form-field__radio m-form-field__lg-target">
+                                <input class="a-radio" type="radio" name="method" id="read-self" value="read-self"
+                                data-gtm-action="Questionnaire Radio Button Clicked"
+                                data-gtm-label="Select how you completed the questionnaire: I read and answered the questions myself"
+                                data-gtm-category="Financial Well-Being Tool Interaction" checked="">
+                                <label class="a-label" for="read-self">
+                                    I read and answered the questions myself
+                                </label>
+                            </div>
+                        </li>
+                        <li class="content-l_col content-l_col-1-2">
+                            <div class="m-form-field m-form-field__radio m-form-field__lg-target">
+                                <input class="a-radio" type="radio" name="method" id="read-to-me" value="read-to-me"
+                                data-gtm-action="Questionnaire Radio Button Clicked"
+                                data-gtm-label="Select how you completed the questionnaire: I read the questions to someone else and recorded their answers"
+                                data-gtm-category="Financial Well-Being Tool Interaction">
+                                <label class="a-label" for="read-to-me">
+                                    I read the questions to someone else and recorded their answers
+                                </label>
+                            </div>
+                        </li>
+                    </ul>
+                </fieldset>
+            </div>
+        </div>
+        <div class="o-form_group">
+            <input class="submit-quiz a-btn a-btn__disabled"
+                  id="submit-quiz" type="submit" value="Get my score"
+                  data-gtm-action="Questionnaire Submitted"
+                  data-gtm-label="Get my score"
+                  data-gtm-category="Financial Well-Being Tool Interaction"
+                  title="Please answer all questions to get your score">
+        </div>
+    </form>`;
+
+function triggerClickEvent( target ) {
+  var event = document.createEvent( 'Event' );
+  event.initEvent( 'click', true, true );
+  return target.dispatchEvent( event );
+}
+
+function initFwbQuestions( ) {
+  fwbQuestions = require( BASE_JS_PATH + 'apps/financial-well-being/fwb-questions' );
+  fwbQuestions.init();
+}
+
+function fillOutForm() {
```

Comment:
I don't really have a better option, but some may say to minimize/eliminate helper functions in tests since it adds untested code within the test itself and creates dependencies across tests that wouldn't otherwise be there. Anyway, just something to mull over on a Sunday afternoon.

Reply:
I've tried to reduce this to only filling out the existing radios rather than injecting more. About as simple as I could get it.